addons/sale/data/sale_demo.xml

```diff
@@ -49,28 +49,22 @@
 
         <record id="sale_order_line_1" model="sale.order.line">
             <field name="order_id" ref="sale_order_1"/>
-            <field name="name" model="sale.order.line" eval="obj().env.ref('product.product_product_25').get_product_multiline_description_sale()"/>
```

Comment:
Seems unrelated diff ?

Reply:
Forgot to do it in another PR (already merged)